src/Employer/Employer.Web/Views/Part2/ApplicationProcess/ApplicationProcess.cshtml

```diff
@@ -71,7 +56,7 @@
 </div>
 @section footerJS
 {
-    <feature name="@FeatureNames.AllowThroughFaaApplicationMethod">
+    <feature>
```

Comment:
Probably remove this `feature` tag as it is redundant. Forget what I said earlier for reference we can just look back in history. It is intended to track feature/integration toggles on the following page https://skillsfundingagency.atlassian.net/wiki/spaces/RAAV2/pages/394068545/Feature+Toggles